lang/en.txt

```diff
@@ -2960,4 +2960,6 @@ dominate the %s into your service!#
 2472 You feel magic weaken against you.#
 2473 You feel magic bounce off you.#
 
-2499 end#
+2499 HP: %d/%d MP: %d/%d#
```

Comment:
Might make sense to put this closer to the rest of the character sheet strings, e.g. at position 1206 which isn't occupied.